charts/lfx-platform/values.yaml

```diff
@@ -392,6 +401,12 @@ authelia_user_generation:
     - committee_member_1
     - committee_member_2
 
+auth0:
+  enabled: false
+
+anonymous_authenticator:
```

Comment:
anonymous authenticator shouldn't be optional IMO, let's remove this value to avoid misunderstanding